.travis.yml

```diff
@@ -2,17 +2,43 @@ language: csharp
 mono: none
 solution: EFCore-Adapter.sln
 dist: xenial
-dotnet: 3.1
+dotnet: 
+    - 3.1
+
+env: 
+    global:
+        - REPO_COMMIT_AUTHOR=$(git show -s --pretty=format:"%cn")
+        - REPO_COMMIT_AUTHOR_EMAIL=$(git show -s --pretty=format:"%ce")
+        - REPO_COMMIT_MESSAGE=$(git show -s --pretty=format:"%s")
+        - DOTNET_SKIP_FIRST_TIME_EXPERIENCE=true
+        - DOTNET_CLI_TELEMETRY_OPTOUT=true
+        - BUILD_CONFIGURATION=Release
+
+install:
+    - sudo apt-get update
+    - sudo apt-get install apt-transport-https
+    - sudo apt-get update
+    - sudo apt-get install dotnet-runtime-2.1 -y
```

Comment:
Can we use dotnet-runtime-3.1?

Reply:
I try to use it and failed because of the tool [coveralls.net](https://github.com/csMACnz/coveralls.net) target dotnet-core-2.1.